extensions-core/protobuf-extensions/pom.xml

```
@@ -45,6 +46,17 @@
     </repository>
   </repositories>
 
+  <dependencyManagement>
+    <dependencies>
+      <!-- This is an indirect dependency of kafka-protobuf-provider
+      update to address vulnerability in transitive dependency okio  -->
+      <dependency>
+          <groupId>com.squareup.okio</groupId>
+          <artifactId>okio</artifactId>
+          <version>3.4.0</version>
```

since you go through the trouble of declaring a property for it, let's use it
```suggestion
          <version>${okio.version}</version>
```